### PR TITLE
Cicd/bitnami dependencies

### PR DESCRIPTION
**Pull request by nicoprow**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,7 +81,6 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>${postgres.version}</version>
             <scope>runtime</scope>
         </dependency>
```
Comment (pazepaze) [inline pom.xml:85]: Did you mean to remove this version? If yes, shouldn't maven property postgres.version also be removed?

File: helm/bpdm/Chart.yaml [hunk 2]
```diff
@@ -24,14 +24,14 @@ version: 1.0.0
...
+    version: 17.*.*
+    repository: https://charts.bitnami.com/bitnami
     alias: elastic
     condition: elastic.enabled
-  - name: bpdm-postgres
-    version: 1.0.0
-    repository: file://charts/bpdm-postgres
+  - name: postgresql
+    version:  11.*.*
+    repository: https://charts.bitnami.com/bitnami
```
Comment (pazepaze) [inline helm/bpdm/Chart.yaml:34]: Why 11?

**Review by pazepaze — CHANGES_REQUESTED** [review]

File: README.adoc [hunk 3]
```diff
@@ -198,33 +218,54 @@ image:
...
-Be aware that additional profiles usually require secrets to be passed to the application.
-The helm deployment automatically creates Kubernetes applicaton secrets which are being used by the deployed application based on the `applicationSecrets` values.
-You can determine which secrets should be created by specifying the name of the entry in the Kubernetes secret with it's corresponding environment variable name (defined in the Spring profiles) and the actual value of the secret like so:
+=== Secrets
+
+Be aware that you may also want to replace placeholder passwords for the deployment.
+This can be done by giving a values file  overriding the password placeholders like so: `helm install release_name ./helm/bpdm -n your_namespace -v secret-values.yaml`
 
+secret-values.yaml:
 [source,yaml]
```
Comment (pazepaze) [inline README.adoc:243]:
```suggestion
secret-values.yaml:

[source,yaml]
```

Comment (nicoprow) [reply README.adoc]: Implemented

File: helm/bpdm/Chart.yaml (hunk 2 above)
Comment (nicoprow) [reply helm/bpdm/Chart.yaml:34]: Is helm chart version but uses 14.2. Updated docker-compose and doku to reflect that

File: pom.xml (hunk 1 above)
Comment (nicoprow) [reply pom.xml]: Added version again

**Review by pazepaze — APPROVED** [review]

**Pull request merged**